zcash_client_sqlite/src/lib.rs

```diff
@@ -587,6 +591,18 @@ impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters> WalletRead for W
             &address.encode(&self.params),
         )
     }
+
+    fn transaction_data_requests(&self) -> Result<Vec<TransactionDataRequest>, Self::Error> {
+        let iter = std::iter::empty();
+
+        #[cfg(feature = "transparent-inputs")]
+        let iter = iter.chain(
+            wallet::transparent::transaction_data_requests(self.conn.borrow(), &self.params)?
+                .into_iter(),
+        );
+
+        Ok(iter.collect())
```

Comment:
```suggestion
        #[cfg(not(feature = "transparent-inputs"))]
        let reqs = vec![];

        #[cfg(feature = "transparent-inputs")]
        let reqs = wallet::transparent::transaction_data_requests(self.conn.borrow(), &self.params)?
            .into_iter(),
            .collect();

        Ok(reqs)
```
Non-blocking.

Reply:
This must have been a victim of a bad rebase; the ordinary request for enhancement and status information is missing. :stuck_out_tongue: